servers/visual_server.cpp

```diff
@@ -617,6 +617,7 @@ Error VisualServer::_surface_set_data(Array p_arrays, uint32_t p_format, uint32_
 			} break;
 			case VS::ARRAY_INDEX: {
 
+				ERR_FAIL_NULL_V(iw, ERR_INVALID_DATA);
```

Comment:
That also seems to be a valid bug, `iw` is only set at line 338 when `r_index_array.size() > 0`.